### PR TITLE
delete code that was copied from the topology-grapher

### PR DESCRIPTION
**Pull request by AndreaCrotti**
Now the topology-grapher is also OSS, if we want to help allow people
to graph their topologies easily they can simply do that adding another
dependency to their dev profiles.

# Checklist

- [ ] tests
- [ ] updated [CHANGELOG](../CHANGELOG.md) (the "unreleased" section)

### CODE REVIEW

**Comment by AndreaCrotti** [comment]
Might be good to merge this first https://github.com/FundingCircle/jackdaw/pull/221
so there is a reference about how do this when it's deleted from here.

File: dev/jackdaw/repl.clj [hunk 1]
```diff
@@ -115,21 +114,6 @@
   given with EDN serdes and a partition count of one."
   (FakeTopicMetadata.))
 
-(defn topology->topic-metadata
```
Comment (creese) [inline dev/jackdaw/repl.clj]: ☝️ This function infers the Kafka topics needed by a topology. We can't simply delete it as this will break examples.

Comment (AndreaCrotti) [reply dev/jackdaw/repl.clj]: Ah sorry I thought this ns was only used in the repl.
Can we just add the grapher as dependency to the examples?

Comment (creese) [reply dev/jackdaw/repl.clj]: That is one option.

The namespace is referenced here:
https://github.com/FundingCircle/jackdaw/blob/master/examples/word-count/dev/user.clj#L32

Comment (AndreaCrotti) [reply dev/jackdaw/repl.clj]: Ok thanks I'll fix the examples then

**Pull request closed without merging**